snapcraft/cli/_options.py

```diff
@@ -130,6 +130,16 @@ def __repr__(self):
         envvar="https_proxy",
         supported_providers=["host", "lxd", "managed-host", "multipass"],
     ),
+    dict(
+        param_decls="--install-ca-certificates",
```

Comment:
can we rename this to "add"

Reply:
We could. I'm not sure what the difference is.  How about just "--ca-certificates"?

Reply:
the guidelines we follow is that install is for heavy operations and add is for simpler resource handling, like this

Reply:
For future reference, are those guidelines documented somewhere?

Reply:
Done